docker-compose.prod.yml

```diff
@@ -15,11 +15,11 @@ services:
 
 #  Redis is used to support sending to Google Analytics events.
```

Comment:
Should we update this comment then?  Maybe "Redis is used for various purposes like Google Analytics events?"...  Or maybe, we should just go ahead and remove these comments altogether.....?

Reply:
I think you can remove it as you said that more components in future might depend on it.